src/components/fieldset/fieldset.html

```diff
@@ -1,6 +1,38 @@
-<section>
-  <fieldset></fieldset>
-</section>
+<form>
+  <section>
+    <p>
+      <label>
+        <span>Enter Task</span>
+        <abbr title="required">*</abbr>
+      </label>
+      <input type="text" id="task" name="task" />
+    </p>
```

Comment:
I wouldn't recommend using a `<p>` here the only thing you need is the `label` and its `input`. However, `input` should be a child element of `label`.